.github/CONTRIBUTING.md

```diff
@@ -8,10 +8,10 @@ The following is a set of guidelines for contributing to the repository. These a
 
 Generally speaking questions are better suited in our resources below.
 
-- The official support server: https://discord.gg/r3sSKJJ
+- The official support server: https://discord.gg/ZebatWssCB
 - The Discord API server under #python_discord-py: https://discord.gg/discord-api
-- [The FAQ in the documentation](https://discordpy.readthedocs.io/en/latest/faq.html)
-- [StackOverflow's `discord.py` tag](https://stackoverflow.com/questions/tagged/discord.py)
+- [The FAQ in the documentation](https://nextcord.readthedocs.io/en/latest/faq.html)
+- [StackOverflow's `nextcord` tag](https://stackoverflow.com/questions/tagged/nextcord)
```

Comment:
I think we should leave the discord.py tag alone

Reply:
I was thinking about that. Maybe mention both?

Reply:
Good idea.